pkg/ephemeral/fake_spdz_test.go

```diff
@@ -93,14 +94,14 @@ func (f *FakePlayer) PublishEvent(name, topic string, event *pb.Event) {
 type FakeExecutor struct {
 }
 
-func (f *FakeExecutor) CallCMD(cmd []string, dir string) ([]byte, []byte, error) {
+func (f *FakeExecutor) CallCMD(theContext context.Context, cmd []string, dir string) ([]byte, []byte, error) {
```

Comment:
The parameter for `context.Context` is usually named `ctx` - at least within the ephemeral code base.
I'd suggest to keep this consistent

Reply:
Done.